src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -372,6 +386,20 @@ protected override ProviderInfo Start(ProviderInfo providerInfo)
                         s_tracer.WriteLine("Not setting home directory {0} - does not exist", homeDirectory);
                 }
             }
+           
+            // OneDrive placeholder support (issue #8315)
+            // make it so OneDrive placeholders are perceived as such with *all* their attributes accessible
+#if !UNIX
+            Version minBuildForPlaceHolderAPIs = new Version(10, 0, 17134, 0);
```

Comment:
It would be good to have an explicit comment on top of this line like `// The ProcessPlaceholderCompatibilityMode APIs are only available on Win10 build 17134 or newer`.